readthedocs/doc_builder/environments.py

```diff
@@ -421,7 +421,11 @@ def update_build(self, state=None):
                 self.build['error'] = str(self.failure)
             else:
                 self.build['error'] = ugettext_noop(
-                    "An unexpected error occurred")
+                    "A failure in our code has occured. "
+                    "Please report this to us with your build id ({}).".format(
+                        self.build['pk']
```

Comment:
Not sure best way to handle this. Interpolation on localized strings should happen outside `gettext`. The interpolated variable should be named as well.